frontend/public/components/modals/language-preferences-modal.tsx

```diff
@@ -10,13 +10,16 @@ import {
   ModalSubmitFooter,
   ModalTitle,
 } from '../factory/modal';
-import { FALLBACK_LOCALE, SUPPORTED_LOCALES } from '../../i18next-parser.config';
 
 const LanguagePreferencesModal = (props: LanguagePreferencesModalProps) => {
   const { i18n, t } = useTranslation();
-  const langOptions = Object.keys(SUPPORTED_LOCALES).map((lang) => ({
+  const supportedLocales = {
+    en: 'English',
+    ja: '日本語',
+  };
+  const langOptions = Object.keys(supportedLocales).map((lang) => ({
```

Comment:
Instead of using a map and transforming it, just initial the variable in the format we need.

Reply:
We're also using supportedLocales as-is to fill the dropdown on line 45 though.